src/components/Generator/GeneredTaskfile/addons/appUtilities/app-utilities.sh

```diff
@@ -0,0 +1,6 @@
+function app:ensure {
+    if [ ! $(which "$1") ]; then
+		echo "Missing required application ${RED}$1${RESET}. Install it before tying again."
+		exit 1
+    fi
```

Comment:
Looks good, as an suggestion/addition when app is found; Show "All set, ${GREEN}$1${RESET} is installed correctly: You're good to go!"